pkg/chartutil/create.go

```diff
@@ -216,10 +216,13 @@ const defaultIgnore = `# Patterns to ignore when building packages.
 const defaultIngress = `{{- if .Values.ingress.enabled -}}
 {{- $fullName := include "<CHARTNAME>.fullname" . -}}
 {{- $svcPort := .Values.service.port -}}
-{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+{{- $capabilities := .Capabilities -}}
+{{- if semverCompare "<1.14-0" $capabilities.KubeVersion.GitVersion -}}
+apiVersion: extensions/v1beta1
+{{- else if and (semverCompare ">=1.14-0" $capabilities.KubeVersion.GitVersion) (semverCompare "<1.19-0" $capabilities.KubeVersion.GitVersion) -}}
```

Comment:
This can be compressed to `else if semverCompare "~1.18-0"`. There's no need for the `semverCompare ">=1.14-0"`: the prior `if` branch will cover all other cases.